### PR TITLE
III-5649 document missing contributors property

### PR DESCRIPTION
**Pull request by corneelwille**
### Added
- contributors property on event model
- contributors property on place model
- contributors property on organizer model

---

Ticket: https://jira.uitdatabank.be/browse/III-5469

### CODE REVIEW

**Review by bertramakers — COMMENTED** [review]
You've added schema files that describe the `contributors` property with titles/descriptions for events/places/organizers, but you still need to add them on the actual models for events/places/organizers:

- https://github.com/cultuurnet/apidocs/blob/main/projects/uitdatabank/models/event.json
- https://github.com/cultuurnet/apidocs/blob/main/projects/uitdatabank/models/place.json
- https://github.com/cultuurnet/apidocs/blob/main/projects/uitdatabank/models/organizer.json

If you want to also add that property to the "read" examples of events/places/organizers, you can add them here:

- https://github.com/cultuurnet/apidocs/blob/main/projects/uitdatabank/models/event-with-read-example.json
- https://github.com/cultuurnet/apidocs/blob/main/projects/uitdatabank/models/place-with-read-example.json
- https://github.com/cultuurnet/apidocs/blob/main/projects/uitdatabank/models/organizer-with-read-example.json

The reason that the examples are not included in the models linked above is that we want to have different examples for "read" and "write" contexts, so that read-only properties are not included in examples of creating/updating an event/place/organizer and vice-versa.

**Comment by corneelwille** [comment]
Thanks for the instructions! 🙏

File: projects/uitdatabank/models/event.json [hunk 1]
```diff
@@ -112,6 +112,9 @@
         },
         "creator": {
           "$ref": "./event-creator.json"
+        },
+        "contributors": {
+        "$ref": "./event-contributors.json"
```
Comment (bertramakers) [inline projects/uitdatabank/models/event.json:117]: Nit: Small indentation issue here

File: projects/uitdatabank/models/place.json [hunk 2]
```diff
@@ -100,6 +100,9 @@
         },
         "sameAs": {
           "$ref": "./place-sameAs.json"
+        },
+        "contributors": {
+        "$ref": "./place-contributors.json"
```
Comment (bertramakers) [inline projects/uitdatabank/models/place.json:105]: Nit: Small indentation issue here

**Pull request merged**